src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+function getNumber(str) {
+  return +str.slice(1).replaceAll(',', '');
+}
+
+const info = document.querySelector('tbody');
+
+document.body.addEventListener('click', e => {
+  const sort = [...info.children].sort((a, b) => {
+    switch (e.target.innerText) {
+      case ('Salary'):
+        let first = getNumber(a.children[3].innerText);
+        let second = getNumber(b.children[3].innerText);
+
+        return first - second;
+
+      case ('Age'):
+        first = +a.children[2].innerText;
+        second = +b.children[2].innerText;
+
+        return first - second;
+
+      case ('Position'):
+        first = a.children[1].innerText;
+        second = b.children[1].innerText;
+
+        return first.localeCompare(second);
+
+      case ('Name'):
+        first = a.children[0].innerText;
+        second = b.children[0].innerText;
+
+        return first.localeCompare(second);
```

Comment:
Look, you use the same methods here. You can combine them. Just create a variable for cell index before.  Use `e.target.cellIndex`.